compio-buf/src/lib.rs

```diff
@@ -0,0 +1,70 @@
+#![cfg_attr(feature = "allocator_api", feature(allocator_api))]
+#![cfg_attr(feature = "read_buf", feature(read_buf))]
+
+//! Utilities for working with buffers.
+//!
+//! Completion APIs require passing ownership of buffers to the runtime. The
+//! crate defines [`IoBuf`] and [`IoBufMut`] traits which are implemented by
+//! buffer types that respect the safety contract.
+
+#[cfg(feature = "arrayvec")]
+pub use arrayvec;
```

Comment:
Do we need doc no inline?

Reply:
I also wonder the reason to reexport them.

Reply:
> I also wonder the reason to reexport them.

It's just best practice to export some optional dependency you implement your trait on

Reply:
> Do we need doc no inline?

![image](https://github.com/compio-rs/compio/assets/38549822/39a24eea-a294-43d5-8d12-9be131fa2b46)
Seems not